### PR TITLE
[13.0][MIG] account_move_line_purchase_info

### PR DESCRIPTION
**Pull request by AdriaGForgeFlow**
Standard migration of module account_move_line_purchase_info to v13.

cc ~ @ForgeFlow

### CODE REVIEW

File: account_move_line_purchase_info/__manifest__.py [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
+#   (https://www.forgeflow.com)
 # License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
 
 {
     "name": "Account Move Line Purchase Info",
     "summary": "Introduces the purchase order line to the journal items",
-    "version": "12.0.2.0.0",
-    "author": "Eficent, " "Odoo Community Association (OCA)",
+    "version": "13.0.1.0.0",
+    "author": "ForgeFlow, " "Odoo Community Association (OCA)",
```
Comment (LoisRForgeFlow) [inline account_move_line_purchase_info/__manifest__.py]: can you remove the double quotes in the middle?

File: account_move_line_purchase_info/models/account_move.py [hunk 2]
```diff
@@ -9,13 +9,6 @@ class AccountMoveLine(models.Model):
 
     _inherit = "account.move.line"
 
-    purchase_line_id = fields.Many2one(
```
Comment (LoisRForgeFlow) [inline account_move_line_purchase_info/models/account_move.py]: already in core now?

**Review by LoisRForgeFlow — COMMENTED** [review]
looks good, a minor comment and a question

File: account_move_line_purchase_info/models/account_move.py (hunk 2 above)
Comment (AdriaGForgeFlow) [reply account_move_line_purchase_info/models/account_move.py]: Yes, it is now in Odoo core

File: account_move_line_purchase_info/__manifest__.py (hunk 1 above)
Comment (AdriaGForgeFlow) [reply account_move_line_purchase_info/__manifest__.py]: Done :+1:

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM :+1:

**Review by AaronHForgeFlow — APPROVED** [review]
:+1: Tested on local LGTM :medal_sports:

**Pull request merged**